react-form-controls/app-name/src/RegistrationFormControlled.tsx

```diff
@@ -1,25 +1,21 @@
 import { useState } from 'react';
 
 export default function RegistrationFormControlled() {
-  const [userName, setUserName] = useState('');
+  const [username, setUserName] = useState('');
```

Comment:
I guess if you changed the case on the variable, you should change it on the setter, also.